### DISCUSSION TITLE
LRO-NAC image exposures from sfs

### DISCUSSION THREAD

**Discussion by mkbarker** [NeoGeographyToolkit/StereoPipeline#405] [Q&A]
Hello,

Could someone elaborate on the meaning and derivation of image exposures by sfs for LRO-NAC images? e.g., how are they computed, what units are they given in, what is the "Local exposure" and how/why is it different from the exposure for the whole image?

The manual says to use --compute-exposures-only "once for a big DEM, before using these for small sub-clips without recomputing them." That implies that there is only one "image exposure" associated with each image and that the big DEM should be bigger than the images in order to compute the correct exposure. Is that true? If so, then would the computed exposure depend on the big DEM's resolution? I'm running some tests on some polar NAC images and the computed exposures do seem to change depending on the big DEM's resolution. 

Thank you,
Mike

**Comment by oleg-alexandrov** [comment]
The exposure value for a camera is computed as the average of the
image intensity divided by the average reflectance, both computed over the
DEM. Later, if the problem is parallelized using ``parallel_sfs``, the
exposures are found just once, over the entire DEM, rather than for each
tile.

For what these mean, see
https://stereopipeline.readthedocs.io/en/latest/sfs_usage.html#mathematical-model.
(I just added the above blurb there about this.)

Indeed, the exposure will depend on DEM resolution, and in fact, on the
precise DEM values. In my experiments, the value should not change much if
your images have big well-lit portions, and if the DEM is reasonable.

The exposure values affect the steepness of the slopes. This is one of
those things we did not spend enough time qualifying, and in fact, we
suspect that the local slopes are somewhat underestimated by SfS.

**Comment by oleg-alexandrov** [comment]
To add, the local exposure is the exposure for a given little tile. That
one is only used to determine if in the current tile there are valid pixels
from the given image, and hence if to use the given image for the given
tile. The pre-computed exposure is used for all tiles in the actual
computation.
